.ci/Jenkinsfile

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env groovy
-@Library('apm@current') _
+@Library('apm@feature/tap-with-multiple-entries-for-nodejs-specific') _
```

Comment:
This should be changed to `current`. Though, still pending to be merged the required PR in the apm-pipeline-library and a new release. 

In any case, I think this particular PR is ready to be reviewed.